### DISCUSSION TITLE
https://www.inven.co.kr/ 접속불가

### DISCUSSION THREAD

**Discussion by swbaek81** [List-KR/community#53] [AdGuard]
윈도우10 엣지브라우저 애드가드윈도우(7.16.0) 인데요

DPI로 부터 보호 체크하면 인벤 접속이 안됩니다. 

DPI로 부터 보호 체크Off 하면 OK 입니다

https://www.inven.co.kr/

![image](https://github.com/List-KR/community/assets/15683789/83b56820-4f79-4def-ad46-494b5a3b91d9)

![image](https://github.com/List-KR/community/assets/15683789/534eeb1e-cb71-424e-98fc-d8e45be8d342)

**Comment by piquark6046** [comment]
DPI 관련해서 AdGuard 고급 설정을 만지셨나요?

**Reply by piquark6046** [reply]
기본 값에서도 그러네요. 잠시만요.

**Comment by piquark6046** [comment]
AdGuard Base 필터를 비활성화하신 환경에서는 아래 규칙을 사용해주세요:
```adblock
@@||inven.co.kr^$stealth=dpi
```

**Reply by piquark6046** [reply]
https://github.com/AdguardTeam/AdguardFilters/commit/e91a049c0caa8457515859c43d760b167434a8cf 고쳤습니다.
